--- ../playscript-monodevelop/main/src/addins/ICSharpCode.NRefactory.PlayScript/ICSharpCode.NRefactory.PlayScript.csproj	2013-05-21 16:14:02.000000000 -0700
+++ ./ICSharpCode.NRefactory.PlayScript/ICSharpCode.NRefactory.PlayScript.csproj	2013-05-21 21:41:33.000000000 -0700
@@ -546,10 +546,9 @@
     <Compile Include="Properties\GlobalAssemblyInfo.cs" />
   </ItemGroup>
   <ItemGroup>
-    <ProjectReference Include="..\..\..\external\nrefactory\ICSharpCode.NRefactory\ICSharpCode.NRefactory.csproj">
-      <Project>{3B2A5653-EC97-4001-BB9B-D90F1AF2C371}</Project>
-      <Name>ICSharpCode.NRefactory</Name>
-    </ProjectReference>
+    <Reference Include="ICSharpCode.NRefactory">
+      <HintPath>..\dependencies\monodevelop\bin\ICSharpCode.NRefactory.dll</HintPath>
+    </Reference>
   </ItemGroup>
   <Import Project="$(MSBuildBinPath)\Microsoft.CSharp.Targets" />
   <ItemGroup />
