--- ../playscript-monodevelop/main/src/addins/ICSharpCode.NRefactory.PlayScript/ICSharpCode.NRefactory.PlayScript.csproj	2013-04-30 17:42:44.000000000 -0700
+++ ./ICSharpCode.NRefactory.PlayScript/ICSharpCode.NRefactory.PlayScript.csproj	2013-05-02 22:02:26.000000000 -0700
@@ -521,10 +521,9 @@
     <Compile Include="Parser\mcs\ps-tokenizer.cs" />
   </ItemGroup>
   <ItemGroup>
-    <ProjectReference Include="..\..\..\external\nrefactory\ICSharpCode.NRefactory\ICSharpCode.NRefactory.csproj">
-      <Project>{3B2A5653-EC97-4001-BB9B-D90F1AF2C371}</Project>
-      <Name>ICSharpCode.NRefactory</Name>
-    </ProjectReference>
+    <Reference Include="ICSharpCode.NRefactory">
+      <HintPath>..\..\..\..\..\..\Applications\Xamarin Studio.app\Contents\MacOS\lib\monodevelop\bin\ICSharpCode.NRefactory.dll</HintPath>
+    </Reference>
   </ItemGroup>
   <Import Project="$(MSBuildBinPath)\Microsoft.CSharp.Targets" />
   <ItemGroup>
